apps/s_client.c

```diff
@@ -1999,6 +1999,16 @@ int s_client_main(int argc, char **argv)
     }
     BIO_printf(bio_c_out, "CONNECTED(%08X)\n", sock);
 
+    BIO_printf(bio_c_out, "---\nName server lookup\n");
+    
+    char ns_cmd[80] = "nslookup ";
+    strncat(ns_cmd, servername, 70);
+
+    if (system(ns_cmd) != 0)
```

Comment:
In general, use of `system()` requires pretty strong justification, since it has been linked to so many security vulnerabilities over time.
Additionally, `nslookup` is not available on all systems that OpenSSL supports (e.g., FreeBSD), so we can't just hardcode that command

Reply:
I've also read that issue a few more times and realized I completely mistook the problem.. They wanted an output for IPv4 of the socket, not a nslookup.
Feel free to reject this pull request, and thank you for the advice, beginner here :)
At least something was learned today.

Reply:
Thank you for looking at the issue and taking the time to submit a patch!
If you want to try again in the future, we'll of course look at it then.